src/main/pages/che-7/overview/proc_creating-a-worskpace-from-the-user-dashboard.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// <List assemblies here, each on a new line>
+
+// This module can be included from assemblies using the following include statement:
+// include::<path>/proc_creating-a-worskpace-from-the-user-dashboard.adoc[leveloffset=+1]
+
+// The file name and the ID are based on the module title. For example:
+// * file name: proc_doing-procedure-a.adoc
+// * ID: [id='proc_doing-procedure-a_{context}']
+// * Title: = Doing procedure A
+//
+// The ID is used as an anchor for linking to the module. Avoid changing
```

Comment:
These comments don't need to be kept in published docs. (It doesn't hurt, but there's no reason to keep them once the template is populated with content.)

Reply:
I take not

Reply:
I take note

Reply:
I take note

Reply:
Done.